test/general/sycl_policy.pass.cpp

```diff
@@ -50,20 +50,70 @@ void test_policy_instance(const Policy& policy)
 int32_t
 main()
 {
+    using namespace oneapi::dpl::execution;
+    static_assert(is_execution_policy<sequenced_policy>::value, "wrong result for is_execution_policy<sequenced_policy>");
```

Comment:
Now host policies are tested in `sycl_policy` test. Would it be better to rename it `policy`? Should we extend `test_policy_instance` for host policies?

Testing of host policies are hidden by `TEST_DPCPP_BACKEND_PRESENT`.

Reply:
Renaming makes sense. I'd rename it to `test_policy`. Regarding extending it doesn't makes sense since we test with `seq`, `unseq`, `par_unseq`, `par` in other tests. The goal of `test_policy_instance` is testing with custom created policies that is not covered by other tests.

>Testing of host policies is hidden by TEST_DPCPP_BACKEND_PRESENT macro.

I don't see where. Could you please point me out?

Reply:
Sorry, `TEST_DPCPP_BACKEND_PRESENT` does not wrap the whole test, but only 26-48 lines. That was a wrong note.